tests/test_headful.py

```diff
@@ -0,0 +1,110 @@
+"""
+Tests relating to headful mode
+"""
+import os
+import platform
+import pytest
+import tempfile
+import shutil
+
+from pyppeteer import launch
+from syncer import sync
+
+EXTENSION_PATH = os.path.join(os.path.dirname(os.path.realpath(__file__)), "assets", "simple-extension")
```

Comment:
We already have an `assets` fixture which provides the path to it, so I think this would be a little more consistent:
```suggestion
@pytest.fixture
def extension_path(assets):
     return assets / "simple-extension"
```
Of course, the rest of the tests will need to be refactored to reflect this change.
```

Reply:
Did not notice it. Will replace it.